doc/examples/README.txt

```diff
@@ -4,4 +4,6 @@ General examples
 -------------------
 
 General-purpose and introductory examples for the scikit.
+A guide_ introduces conventions and basic image manipulations.
```

Comment:
line break between the two sentences?

I would suggest a slight change 

The <user guide>_ (narrative documentation) introduces conventions and basic image manipulations.